karma.conf.js

```diff
@@ -23,43 +22,13 @@ module.exports = function(config) {
     // preprocess matching files before serving them to the browser
     // preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors: {
-      //'tests/*.js': ['webpack', 'babel', 'sourcemap']
       'tests/*.js': ['webpack', 'sourcemap']
     },
 
     webpack: {
       //mode: 'production',
       mode: 'development',
-      devtool: 'inline-source-map',
-      module: {
-        rules: [
-          /*
-          {
-            test: /\.js$/,
-            include: [{
-              // exclude node_modules by default
-              exclude: /(node_modules)/
-            }],
-            use: {
-              loader: 'babel-loader',
-              options: {
-                presets: ['@babel/preset-env'],
-                plugins: [
-                  '@babel/plugin-transform-modules-commonjs',
-                  '@babel/plugin-transform-runtime'
-                ]
-              }
-            }
-          }
-          */
-        ]
-      },
-      node: {
-        Buffer: false,
-        process: false,
-        crypto: false,
-        setImmediate: false
-      }
```

Comment:
FYI, be careful when removing these flags.  Sometimes we want to check the code works in the browser when no sub-deps pull in the polyfills for these.  Hard to know when to use them or not which is why they are sprinkled throughout all our repos.

Reply:
@davidlehn Ok, however do we need them in this specific case?

Reply:
Short answer:
Can probably not worry about it until it's a problem.

Long answer:
I'm not sure.  It's easy to say no when there are only simple dependencies.  But here it will indirectly pull in those polyfills.  Tests pass either way.  Sometimes code pulls in the polyfills but doesn't use them.  Other times it could change behavior.  Tracking this all down is not very fun.

To just look at the deps, you can be more verbose with webpack and compare with and without those rules and possibly 'externals' as well:
```
npm i --no-save webpack-cli
./node_modules/.bin/webpack --display-modules --sort-chunks-by=size . | grep -iE 'buffer|process|crypto|setimmediate'
```

Can also play with graphs if you want to dig:
```
./node_modules/.bin/webpack --json . > stats.json
https://github.com/webpack/analyse
```

Adding those rules in would better mimic what we do at the top level.  But can probably just not worry about it until it's a problem.  Over time we should update our libs to not even pull in polyfills if possible.